.travis.yml

```diff
@@ -1,4 +1,20 @@
-language: java
+language: android
 jdk:
   - openjdk7
   - oraclejdk8
+env:
+  - TEST=java
+  - TEST=android
+matrix:
+  exclude:
+    - jdk: oraclejdk8
+      env: TEST=android
+android:
+  components:
+    - android-15
+    - build-tools-21.1.1
```

Comment:
The [sys-img changes](https://github.com/atlassian/commonmark-java/commit/e5674e85e3005d6a8a465816b80b2ae73010b977) don't work on Travis: https://travis-ci.org/atlassian/commonmark-java/jobs/110851523